spec.html

```diff
@@ -9337,6 +9337,20 @@ <h1>Static Semantics: UTF16Decode ( _lead_, _trail_ )</h1>
         1. Return the code point _cp_.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-codepointat" aoid="CodePointAt">
+      <h1>Static Semantics: CodePointAt ( _string_, _position_ )</h1>
+      <p>The abstract operation CodePointAt interprets a String _string_ as a sequence of UTF-16 encoded code points, as described in <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and reads from it a single code point starting with the code unit at index _position_. When called, the following steps are performed:</p>
```

Comment:
Nit: `UTF-16 encoded` → `UTF-16-encoded`

Reply:
This one I will leave for now, because the "UTF-16 encoded" text is common to several other parts of the spec (e.g., [ToNumber](https://tc39.github.io/ecma262/#sec-tonumber-applied-to-the-string-type) and many subsections of [String objects] (https://tc39.github.io/ecma262/#sec-string-objects)).